LibraryManagementSystem/Forms/ReturnTab.cs

```diff
@@ -20,6 +20,8 @@ public ReturnTab()
         SqlConnection con;
         SqlCommand cmd;
         SqlDataAdapter da;
+        DataTable dt;
+        int bookID;
```

Comment:
If there's no issues, create variable *bookID* within the *btnReturn_Click* method to have all the variables grouped together
<kbd>int bookID;
string availText = "Avail";</kbd>